lib/customer.rb

```diff
@@ -0,0 +1,40 @@
+require "CSV"
+
+class Customer
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  attr_reader :id
+
+  attr_accessor :email, :address
```

Comment:
A couple of things

1.  It's standard to put the attr_accessor and attr_reader lines at the top of the class, for readability.
1.  Do you need to give users of the class the ability to directly change the email or address?  These should be readers instead.  Give the user minimal change to change your instance variables to invalid values.